### DISCUSSION TITLE
logging custom errors to sentry?

### DISCUSSION THREAD

**Discussion by gigamesh** [trpc/trpc#5782] [Q&A]
I'm using Sentry. Rather than every error showing up as `TRPCError`, I want to be able to create custom errors so its more easy to distinguish between them and set up alerts for very specific types of errors. ex: `CriticalServiceDownError`

Is this possible? The only way I've been able to do it is to log directly to sentry via Sentry.captureException before it bubbles up to the trpc error handler, but that feels pretty janky.

**Comment by mbifulco** [comment]
I'm also interested in this - I haven't come up with anything that works well yet. All of our errors show up as TRPCError, which makes it hard to tell one Sentry issue from the next (or whether things are recurring / new / regressions etc). 

At a minimum I'd love for the error to have something like: `Error in appRouter.post.get` (etc)
